packages/test-factory/src/factory.test.ts

```diff
@@ -878,4 +878,25 @@ describe('factory', () => {
       expect(factory.user.build().data).toBe(data);
     });
   });
+
+  describe('pick with create (no argument)', () => {
+    it('should only generate values for picked columns in the async branch', async () => {
```

Comment:
this test doesn't really test the change: it's green both before and after the change.